### PR TITLE
Branch attendance v1

### PR DESCRIPTION
**Pull request by Evande1**
1. Made attendance <index> m/<0 or 1> 
1. Checks validation as well. 
1. Haven't made the test cases

### CODE REVIEW

**Comment by Evande1** [comment]
fixed test cases, will introduce more test cases in v1.3

**Comment by Evande1** [comment]
Fixes #57

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -20,6 +21,7 @@
 import seedu.address.commons.util.CollectionUtil;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
+import seedu.address.model.attendance.Attendance;
```
Comment (zzthian) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:24]: Correct order of imports! Nice

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -233,7 +237,7 @@ public boolean equals(Object other) {
             if (!(other instanceof EditStudentDescriptor)) {
                 return false;
             }
-
+            // might get errors here
```
Comment (zzthian) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:240]: Why would this get an error? Might want to explain in the comment

**Review by zzthian — APPROVED** [review]
Everything looks good! Just some commented out code we need to address.

**Comment by Evande1** [comment]
closes #57

**Pull request merged**